src/annotations/api/index.ts

```diff
@@ -1,8 +1,13 @@
-// Libraries
-import axios, {AxiosResponse} from 'axios'
-
 // Types
-import {PostAnnotationResponse, PostAnnotationPayload} from 'src/types'
+import {
+  Annotation,
+  GetAnnotationResponse,
+  GetAnnotationPayload,
+  DeleteAnnotation,
+} from 'src/types'
+
+// Libraries
+import axios from 'axios'
```

Comment:
nit: can we get a comment and a link to the issue that will remove axios once the API is developed?